.travis.yml

```diff
@@ -35,23 +34,23 @@ before_install:
   - export PATH="$HOME/.yarn/bin:$PATH"
 
 install:
-  - yarn install --production=false
+  - yarn install
 
 jobs:
   include:
   - stage: pull_request
     script:
       - >-
-        yarn concurrently
-        "yarn lint"
-        "yarn depcheck"
-        "yarn test"
+        yarn run-p -l
```

Comment:
For some reason, `concurrently` is about 40mb and `npm-run-all/run-p` pretty much do the same thing at a fraction of the size.